module/server.go

```diff
@@ -99,6 +99,11 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	h.ServeHTTP(w, r)
 }
 
+// Stats is unsupported.
+func (s *Server) Stats() any {
```

Comment:
The `rpc.Server` interface now has a `Stats` method that must be implemented; I assume we want to leave this "unsupported" for the module server for now, @dgottlieb.

Reply:
Thanks for flagging this. Is this the server that's part of the module binary? Returning nil is fine for that.

Reply:
> Is this the server that's part of the module binary?

If I understand your question correctly, yes. It's the RPC server that the module registers its own service with to receive `AddResourceRequest`, `ReconfigureResourceRequest`, etc.

Reply:
Cool, makes sense. I think this is fine. I think the other option is to make `module.NewServer` just return its actual type and not [some random interface](https://github.com/viamrobotics/rdk/blob/6d17e174c1b1595c4f70b32848d5c99923d4fbbc/module/server.go#L17)

Reply:
Yeah... current design is a little gross; I suppose we want the `Server` type here to _partially_ fulfill the `rpc.Server` interface (i.e. avoid removal of some key methods.) 🤷🏻 Will leave as-is for now.